jujugui/static/gui/src/app/components/profile/header/_header.scss

```diff
@@ -1,126 +1,13 @@
 .profile-header {
-  background-color: $white;
-  padding-top: 30px;
-  padding-bottom: 30px;
-  border-bottom: 1px solid $light-grey;
+  margin-bottom: 3rem;
 
-  &__inner {
-    display: flex;
+  .p-strip {
```

Comment:
Isn't `p-sprip--light` white.

Reply:
No, it's grey - https://docs.vanillaframework.io/en/patterns/strip